app/src/main/res/layout/activity_forgot_password.xml

```diff
@@ -0,0 +1,164 @@
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="@font/epilogue_bold"
+                    android:text="@string/Password"
+                    android:textColor="@color/blue"
+                    android:textSize="28sp"
+                    app:layout_constraintStart_toStartOf="@+id/tvForgotTitle"
+                    app:layout_constraintTop_toBottomOf="@+id/tvForgotTitle" />
+
+                <!-- Login Fields -->
+
+                <LinearLayout
+                    android:id="@+id/llLoginFields"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginHorizontal="32dp"
+                    android:orientation="vertical"
+                    app:layout_constraintBottom_toTopOf="@+id/btnReset"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/Password">
+
+                <TextView
+                    android:id="@+id/tvLoginEmail"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="@font/epilogue_medium"
+                    android:text="@string/email"
+                    android:textColor="@color/light_black"
+                    android:textSize="15sp" />
+
+                    <EditText
+                        android:id="@+id/etLoginEmail"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:layout_marginTop="8dp"
+                        android:autofillHints="emailAddress"
+                        android:background="@drawable/custom_edittext"
+                        android:elevation="1dp"
+                        android:fontFamily="@font/inter_regular"
+                        android:hint="@string/hint_email"
+                        android:inputType="textEmailAddress"
+                        android:padding="12dp"
+                        android:textColor="@color/black"
+                        android:textColorHint="@color/leaves_txt"
+                        android:textSize="16sp" />
+
+                    <TextView
+                        android:id="@+id/space"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="24dp"
+                        />
+                </LinearLayout>
+
+                <Button
+                    android:id="@+id/btnReset"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="144dp"
```

Comment:
Remove bottom margin and instead add appropriate top margin to adjust the button position